templates/docs/patterns/badge/accessibility.md

```diff
@@ -16,10 +16,15 @@ This component strives to follow [WCAG 2.1 (level AA) guidelines](https://www.w3
 
 - Select an appropriately coloured badge which makes sense semantically for your use case.
 - Ensure you don’t use colour as the only visual means of conveying information or an action.
+- Use aria-label to provide full context for screen readers. For example, "you have 999+ machines".
+- If the badge is used to represent a live status of some value, using role="status" should be considered.
 
 ## Resources
 
 Applicable WCAG guidelines:
 
 - [WCAG21 - Use of color](https://www.w3.org/TR/WCAG21/#use-of-color)
 - [WCAG21 - Sensory characteristics](https://www.w3.org/TR/WCAG21/#sensory-characteristics)
+- [WCAG21 - Role](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22#example-2-updating-the-shopping-cart-status)
+- [Material UI - Badge accessibility](https://mui.com/material-ui/react-badge/#accessibility)
```

Comment:
We don't need to list it, especially in the "Applicable WCAG guidelines" list :)

Reply:
:+1: